<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUI</class>
 <widget class="QMainWindow" name="MainWindowUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="KLed" name="connectedLED">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="state">
         <enum>KLed::Off</enum>
        </property>
        <property name="shape">
         <enum>KLed::Circular</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Connected</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="general_tab">
         <property name="minimumSize">
          <size>
           <width>782</width>
           <height>496</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>782</width>
           <height>496</height>
          </size>
         </property>
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="bottomMargin">
               <number>30</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>devices connected</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTreeWidget" name="listAllDevices">
                <column>
                 <property name="text">
                  <string notr="true">Devices</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="SimpleDesc">
                <property name="text">
                 <string>Request Simple Desc</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button1">
                <property name="text">
                 <string>Annunce device</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button2">
                <property name="text">
                 <string>Syntetic Simple Desc</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="light_device">
         <attribute name="title">
          <string>light devices</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Light devices</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeLightDevices">
              <property name="columnCount">
               <number>1</number>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Light Devices</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="rightMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QTreeWidget" name="bindTable">
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Bind Table</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Bind table</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateBindTableButton">
              <property name="text">
               <string>Update Bind Table</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bindButton">
              <property name="text">
               <string>Bind</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Button Devices</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeButtonDevices">
              <property name="columnCount">
               <number>1</number>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Button devices</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
